app/models/case/sar/offender.rb

```diff
@@ -64,6 +64,8 @@ def close_expired_rejected
         CaseClosureService.new(kase, User.system_admin, {}).call
       end
     end
+
+    AUTOCLOSUREDEADLINE = 90
```

Comment:
We usually use underscores on constants for clarity.
Maybe the following?
```suggestion
    REJECTED_AUTO_CLOSURE_DEADLINE = 90
```

Reply:
Also, it should be with the other constants in the class.